graylog2-server/src/main/java/org/graylog2/alerts/AlertConditionBindings.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.alerts;
+
+import com.google.inject.multibindings.MapBinder;
+import org.graylog2.alerts.types.FieldContentValueAlertCondition;
+import org.graylog2.alerts.types.FieldValueAlertCondition;
+import org.graylog2.alerts.types.MessageCountAlertCondition;
+import org.graylog2.plugin.alarms.AlertCondition;
+import org.graylog2.plugin.inject.Graylog2Module;
+
+public class AlertConditionBindings extends Graylog2Module {
+    @Override
+    protected void configure() {
+        MapBinder<String, AlertCondition.Factory> alertConditionBinder = alertConditionBinder();
+        installAlertConditionWithCustomName(alertConditionBinder,
```

Comment:
Please add a `addAlertCondition()` (or similar) method to `PluginModule` like we have for alarm callbacks and others. You can then also just extend `PluginModule` and use that method in here.